src/structures/command/Command.js

```diff
@@ -148,4 +148,18 @@ module.exports = class Command {
       subcommands
     }
   }
+
+  getEmoji (emoji) {
```

Comment:
Why not use a loader? The function receives emoji guilds, get emojis and put its on store, for example, `client.officialEmojis`, so using `this.client.officialEmojis.get('EMOJI_NAME')` to get a emoji, or keep `getEmoji` for "short name".

Reply:
^

Reply:
i mean yeah that can be doable